tensorflow/contrib/learn/python/learn/datasets/BUILD

```diff
@@ -47,7 +47,10 @@ filegroup(
 py_test(
     name = "load_csv_test",
     size = "small",
-    srcs = ["load_csv_test.py"],
+    srcs = [
+        "load_csv_test.py",
+        "synthetic_test.py"
```

Comment:
can you please split this into two tests in build file?
it's preferred to have tests separate to allow better parallelism.